cmd/end2endtest/main.go

```diff
@@ -31,39 +23,26 @@ import (
 // * how many times to retry opening a connection to an endpoint before giving up
 const retries = 10
 
+type VochainTest interface {
+	Setup(api *apiclient.HTTPclient, config *config) error
+	Run() (duration time.Duration, err error)
```

Comment:
Document that duration is expected to be the actual time of the test, without considering any setup phase.

Reply:
done. i actually refactored the code so that duration is calculated by the caller, makes everything cleaner. I also now report the duration of the Setup and Teardown steps, which might be relevant